specs/reservation_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'spec_helper'
+
+describe "Reservation class" do
+
+  before do
+    start_date = Date.new(2018,5,20)
+    end_date = start_date + 4
+    @reservation_data = {
```

Comment:
Don't make it an `@instance_variable` unless you really really need it somewhere else.